packages/node_modules/nav-frontend-lesmerpanel/src/lesmerpanelToggle.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+import Chevron from 'nav-frontend-chevron';
+
+export interface LesMerToggleProps {
+    erApen: boolean;
+    onClick: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+    lukkTekst: React.ReactNode;
+    apneTekst: React.ReactNode;
+}
+
+class LesMerToggle extends React.Component<LesMerToggleProps> {
+    render() {
+        const { erApen, onClick, lukkTekst, apneTekst } = this.props;
+        const btnClassName = cn('lesMerPanel__togglelink', {
+            'lesMerPanel__togglelink--erApen': erApen
+        });
+        return (<div className="lesMerPanel__toggle">
+            <button
```

Comment:
Bør legge inn en eksplisitt `role="button"` slik at komponenten kan brukes inne i skjemaer uten fare for at knappen får implisitt `role="submit"`.